<launch>

    <node pkg="world_cloud_converter" type="cloud_converter_cuda_r1" name="" output="screen">
        <param name="ed_left" value="7.7175"/>
        <param name="ed_right" value="-0.2825"/>
        <param name="ed_ceil" value="14.7175"/>
        <param name="ed_floor" value="-0.2825"/>
        <param name="ed_resolution" value="0.2"/>
        <param name="half_height" value="-0.25"/>
        <param name="half_width" value="-0.25"/>
        <param name="cloud_topic_1" value="/livox/lidar_192_168_1_3"/>
        <param name="cloud_topic_2" value="/livox/lidar_192_168_1_115"/>
    </node>

</launch>